Geospatial.md

```diff
@@ -0,0 +1,144 @@
+  - under the License.
+  -->
+
+Geospatial Definitions
+====
+
+This document contains the specification of geospatial types and statistics.
+
+# Background
+
+The Geometry and Geography class hierarchy and its Well-Known Text (WKT) and
+Well-Known Binary (WKB) serializations (ISO supporting XY, XYZ, XYM, XYZM) are
+defined by [OpenGIS Implementation Specification for Geographic information –
+Simple feature access – Part 1: Common architecture][sfa-part1], from [OGC
+(Open Geospatial Consortium)][ogc].
+
+The version of the OGC standard first used here is 1.2.1, but future versions
+may also be used if the WKB representation remains wire-compatible.
+
+[sfa-part1]: https://portal.ogc.org/files/?artifact_id=25355
+[ogc]: https://www.ogc.org/standard/sfa/
+
+## Coordinate Reference System
+
+Coordinate Reference System (CRS) is a mapping of how coordinates refer to
+locations on Earth.
+
+The default CRS `OGC:CRS84` means that the objects must be stored in longitude,
+latitude based on the WGS84 datum.
+
+Custom CRS can be specified by a string value. It is recommended to use the
+identifier of the CRS like [Spatial reference identifier][srid] and [PROJJSON][projjson].
+
+For geographic CRS, longitudes are bound by [-180, 180] and latitudes are bound
+by [-90, 90].
+
+[srid]: https://en.wikipedia.org/wiki/Spatial_reference_system#Identifier
+[projjson]: https://proj.org/en/stable/specifications/projjson.html
+
+## Edge Interpolation Algorithm
+
+An algorithm for interpolating edges, and is one of the following values:
+
+* `spherical`: edges are interpolated as geodesics on a sphere.
+* `vincenty`: [https://en.wikipedia.org/wiki/Vincenty%27s_formulae](https://en.wikipedia.org/wiki/Vincenty%27s_formulae)
+* `thomas`: Thomas, Paul D. Spheroidal geodesics, reference systems, & local geometry. US Naval Oceanographic Office, 1970.
+* `andoyer`: Thomas, Paul D. Mathematical models for navigation systems. US Naval Oceanographic Office, 1965.
+* `karney`: [Karney, Charles FF. "Algorithms for geodesics." Journal of Geodesy 87 (2013): 43-55](https://link.springer.com/content/pdf/10.1007/s00190-012-0578-z.pdf), and [GeographicLib](https://geographiclib.sourceforge.io/)
+
+# Logical Types
+
+Two geospatial logical type annotations are supported:
+* `GEOMETRY`: Geometry features in the WKB format with linear/planar edges interpolation. See [Geometry](LogicalTypes.md#geometry)
+* `GEOGRAPHY`: Geography features in the WKB format with an explicit (non-linear/non-planar) edges interpolation algorithm. See [Geography](LogicalTypes.md#geography)
+
+# Statistics
+
+`GeometryStatistics` is a struct specific for `GEOMETRY` and `GEOGRAPHY` logical
+types to store statistics of a column chunk. It is an optional field in the
+`ColumnMetaData` and contains [Bounding Box](#bounding-box) and [Geometry
+Types](#geometry-types) that are described below in detail.
+
+## Bounding Box
+
+A geometry has at least two coordinate dimensions: X and Y for 2D coordinates
```

Comment:
Nit: 'geometry' may be misleading as its both geometry and geography.  how about 'A bounding box value has at least...'?